src/portfolio/Footer.js

```diff
@@ -0,0 +1,44 @@
+const Footer = () => {
```

Comment:
Automatic return const Footer = () => <>...</>
No need for a return since you only have html